cachito/web/api_v1.py

```diff
@@ -184,6 +184,70 @@ def get_request(request_id):
     return flask.jsonify(json)
 
 
+def get_latest_request() -> flask.Response:
+    """
+    Retrieve the latest request for a repo_name/ref.
+
+    The latest request will be the one with the highest id.
+
+    :return: a Flask JSON response
+    :rtype: flask.Response
+    :raise NotFound: if the request is not found
+    """
+    # mypy: Connexion ensures that these cannot be None
+    repo_name = cast(str, flask.request.args.get("repo_name"))
+    ref = cast(str, flask.request.args.get("ref"))
+
+    request = _get_latest_request_by_repo_name_and_ref(repo_name, ref)
+
+    if not request:
+        raise NotFound
+
+    return flask.jsonify(request.to_json())
```

Comment:
If we don't set `request.to_json(verbose=False)`, it will return the fully resolved request (with packages and deps). I think we want the shorter version, right?